backend/fixtures/polling_stations.sql

```diff
@@ -1,13 +1,13 @@
 INSERT INTO polling_stations (id, election_id, name, number, number_of_voters, polling_station_type, street,
                               house_number, house_number_addition, postal_code, locality)
-VALUES (1, 1, 'Op Rolletjes', 33, NULL, 'mobiel', 'Rijksweg A12', '1', NULL, '1234 YQ', 'Den Haag'),
-       (2, 1, 'Testplek', 34, 1000, 'bijzonder', 'Teststraat', '2', 'b', '1234 QY', 'Testdorp'),
-       (3, 2, 'Testgebouw', 35, NULL, 'vaste_locatie', 'Testweg', '3', NULL, '1234 QA', 'Teststad'),
-       (4, 3, 'Studio The Rules', 1, NULL, 'vaste_locatie', 'Gerontoplein', '1', NULL, '1337 YQ', 'Juinen'),
-       (5, 3, 'Buurtcentrum de Mattenklopper', 2, 1000, 'bijzonder', 'Complexiteitslaan', '2', 'b', '1337 QY',
+VALUES (1, 1, 'Op Rolletjes', 33, NULL, 'Mobile', 'Rijksweg A12', '1', NULL, '1234 YQ', 'Den Haag'),
+       (2, 1, 'Testplek', 34, 1000, 'Special', 'Teststraat', '2', 'b', '1234 QY', 'Testdorp'),
+       (3, 2, 'Testgebouw', 35, NULL, 'Fixedlocation', 'Testweg', '3', NULL, '1234 QA', 'Teststad'),
+       (4, 3, 'Studio The Rules', 1, NULL, 'Fixedlocation', 'Gerontoplein', '1', NULL, '1337 YQ', 'Juinen'),
```

Comment:
Does `Fixedlocation` instead of `FixedLocation` still work?

Reply:
Fixed the casing in [9aa737d](https://github.com/kiesraad/abacus/commit/9aa737d1033219f23e209d27efc1f33ace78ee46). Haven't tested if it still works though.

Reply:
It doesn't, so the case is covered by the tests. I wonder why it got through CI though?

Reply:
I was wondering this too, I think it's because we don't retrieve the polling stations for this election in our tests.